### PR TITLE
remove sdk realm language tabs

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

remove tabs with sdk links inside step directives. done b/c causes x-overflow issues per https://jira.mongodb.org/browse/DOP-3589

also removed all `.. tabs-realm-languages::` that just contain links to SDK pages, replacing them with a list of SDK links. this is something we've talked about doing in the past. figured i'd get it in while doing related work here.

### Jira

n/a

### Staged Changes

- [App Services (changes throughout)](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/remove_long_tabs_in_steps/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/authentication/custom-jwt.txt [hunk 1]
```diff
@@ -801,55 +801,22 @@ Examples
...
+- To register or log in a Custom JWT user from the .NET SDK, see the
+  :ref:`.NET SDK guide to Custom JWT authentication <dotnet-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Node SDK, see the
+  :ref:`Node.js SDK guide to Custom JWT authentication <node-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the React Native SDK, see the
+  :ref:`React Native SDK guide to Custom JWT authentication <react-native-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Swift SDK, see the
+  :ref:`Swift SDK guide to Custom JWT authentication <ios-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Web SDK, see the
+  :ref:`Web SDK guide to Custom JWT authentication <web-login-custom-jwt>`.
```
Comment (nlarew) [inline source/authentication/custom-jwt.txt:819]: Lots of repetitive text here. Can we pull out the common sentence and just have a bulleted list of links?

File: source/authentication/custom-jwt.txt [hunk 2]
```diff
@@ -801,55 +801,22 @@ Examples
...
-
-      To register or log in a Custom JWT user from the Kotlin Client SDK, see the
-      :ref:`Kotlin SDK guide to Custom JWT authentication <kotlin-login-jwt>`.
-
-   .. tab::
-      :tabid: flutter
-
-      To register or log in a Custom JWT user from the Flutter Client SDK, see the
-      :ref:`Flutter SDK guide to Custom JWT authentication <flutter-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Flutter SDK, see the
```
Comment (cbush) [inline source/authentication/custom-jwt.txt:804]: The repetition was hidden in the tab view, here it is very obvious and kinda distracting. Maybe we should just have the link with one preamble saying "To register ... see the corresponding SDK documentation:"

File: source/authentication/custom-jwt.txt [hunk 3]
```diff
@@ -801,55 +801,15 @@ Examples
...
-
-   .. tab::
-      :tabid: flutter
-
-      To register or log in a Custom JWT user from the Flutter Client SDK, see the
-      :ref:`Flutter SDK guide to Custom JWT authentication <flutter-login-custom-jwt>`.
+- :ref:`Custom JWT Authentication - Flutter SDK<flutter-login-custom-jwt>`.
+- :ref:`Custom JWT Authentication - Java SDK <java-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Kotlin SDK, see the
+  :ref:`Custom JWT Authentication - Kotlin SDK <kotlin-login-jwt>`.
```
Comment (nlarew) [inline source/authentication/custom-jwt.txt:807]:
```suggestion
- :ref:`Custom JWT Authentication - Kotlin SDK <kotlin-login-jwt>`.
```

File: source/sync/data-model/data-model-map.txt [hunk 4]
```diff
@@ -192,42 +192,13 @@ You can configure the following constraints for a given property:
...
-      :tabid: kotlin
-
-      :ref:`kotlin-supported-types`
+:ref:`flutter-data-types`
+:ref:`java-data-types`
+:ref:`kotlin-supported-types`
+:ref:`dotnet-field-types`
+:ref:`node-data-types`
+:ref:`react-nativedata-types-field-types`
+:ref:`Supported Property Types - Swift SDK <ios-supported-property-types>`
```
Comment (nlarew) [inline source/sync/data-model/data-model-map.txt:201]:
```suggestion
- :ref:`flutter-data-types`
- :ref:`java-data-types`
- :ref:`kotlin-supported-types`
- :ref:`dotnet-field-types`
- :ref:`node-data-types`
- :ref:`react-nativedata-types-field-types`
- :ref:`Supported Property Types - Swift SDK <ios-supported-property-types>`
```

File: source/sync/data-model/data-model-map.txt [hunk 5]
```diff
@@ -237,42 +208,13 @@ Both Realm Object Schemas and App Services Schemas support array properties.
...
-      :tabid: kotlin
-
-      :ref:`kotlin-supported-types`
+:ref:`Dart Types - Flutter SDK <flutter-dart-types>`
+:ref:`Lists - Java SDK <java-field-relationships-lists>`
+:ref:`kotlin-supported-types`
+:ref:`Lists - .NET SDK <dotnet-property-lists>`
+:ref:`Supported Property Types - Node.js SDK <node-supported-property-types>`
+:ref:`Supported Property Types - React Native SDK <react-native-supported-property-types>`
+:ref:`swift-supported-types`
```
Comment (nlarew) [inline source/sync/data-model/data-model-map.txt:217]:
```suggestion
- :ref:`Dart Types - Flutter SDK <flutter-dart-types>`
- :ref:`Lists - Java SDK <java-field-relationships-lists>`
- :ref:`kotlin-supported-types`
- :ref:`Lists - .NET SDK <dotnet-property-lists>`
- :ref:`Supported Property Types - Node.js SDK <node-supported-property-types>`
- :ref:`Supported Property Types - React Native SDK <react-native-supported-property-types>`
- :ref:`swift-supported-types`
```

**Review by nlarew — APPROVED** [review]
LGTM after suggested changes

**Pull request merged**